modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -181,12 +180,14 @@ export default {
       autofocus: this.autofocus,
       onUpdate: this.editorUpdate,
       extensions: [
-        StarterKit,
+        StarterKit.configure({
+          document: false,
+          heading: false
+        }),
         TextAlign.configure({
           types: [ 'heading', 'paragraph' ]
         }),
         Highlight,
-        TextStyle,
```

Comment:
Remove `TextStyle` duplicate (already added in `this.aposTiptapExtensions`)

![image](https://user-images.githubusercontent.com/8301962/192538564-594e8e91-a761-4c3f-a4c0-0bf0ec34afdf.png)